applications/services/gui/modules/variable_item_list.c

```diff
@@ -413,6 +597,42 @@ void variable_item_list_process_ok(VariableItemList* variable_item_list) {
         true);
 }
 
+void variable_item_list_process_ok_long(VariableItemList* variable_item_list) {
+    furi_check(variable_item_list);
+
+    with_view_model(
+        variable_item_list->view,
+        VariableItemListModel * model,
+        {
+            VariableItem* item = variable_item_list_get_selected_item(model);
+
+            bool is_allowed = false;
```

Comment:
this is just for prevent mistaken-touch for those two options field. but they works good too, just i think not needed